lib/routes/chinadaily/language.ts

```diff
@@ -0,0 +1,326 @@
+import { type Data, type DataItem, type Route, ViewType } from '@/types';
+
+import { art } from '@/utils/render';
+import cache from '@/utils/cache';
+import { getCurrentPath } from '@/utils/helpers';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+import { type CheerioAPI, type Cheerio, type Element, load } from 'cheerio';
+import { type Context } from 'hono';
+import path from 'node:path';
+
+const __dirname = getCurrentPath(import.meta.url);
+
+export const handler = async (ctx: Context): Promise<Data> => {
```

Comment:
```suggestion
import ofetch from '@/utils/ofetch';
import { parseDate } from '@/utils/parse-date';
import timezone from '@/utils/timezone';

import { type CheerioAPI, type Cheerio, type Element, load } from 'cheerio';
import { type Context } from 'hono';
import path from 'node:path';

export const handler = async (ctx: Context): Promise<Data> => {
```
No longer needed as it will be provided by tsdown when transpiling https://github.com/DIYgod/RSSHub/blob/f19a725157ff7ea4c0bdd04f7708ff387b2926d8/tsdown.config.ts#L7